bigbluebutton-html5/app/client/stylesheets/style.less

```diff
@@ -834,26 +841,32 @@ body {
 
 .settingsMenu {
   height: 100%;
-  .linear-gradient(rgb(65,68,77), rgb(58,60,69));
+  z-index: 1000;
+  position: fixed;
+  transform: translateX(100vw);
+  width: 70vw;
 
-  @media @mobile-portrait-with-keyboard, @mobile-portrait {
-    z-index: 1000;
-    position: fixed;
-    transform: translateX(100vw);
-    width: 60vw;
+  transition: transform 0.3s linear;

```

Comment:
Have you guys thought about using a mixin library, such as http://lesselements.com/? It would help clean up these vendor prefixes (and probably simplify other things as well). This one is apparently good too: http://lesshat.madebysource.com/